app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -108,6 +109,7 @@ private void initRecyclerView() {
         }
 
         rvContributionsList.setAdapter(adapter);
+        rvContributionsList.setItemViewCacheSize(0);
```

Comment:
View recycling is kind of the point of a Recyclerview. I am very hesitant to approve this.

Reply:
Yes yes, of course I was gonna revert this commit(if you see the commit message), just added this to see if this would solve Nes's issue so that we can create a separate issue to discuss how to improve performance

Reply:
Although this does not entirely nullify the recycler view's job

Reply:
No but it is in the name